make/test.mk

```diff
@@ -62,7 +62,7 @@ prepare-and-deploy-e2e: prepare-e2e deploy-e2e
 	@echo "To clean the cluster run 'make clean-e2e-resources'"
 
 .PHONY: verify-migration-and-deploy-e2e
-verify-migration-and-deploy-e2e: prepare-projects e2e-deploy-latest e2e-migration-setup get-publish-and-install-operators e2e-migration-verify
+verify-migration-and-deploy-e2e: check-quay-login-if-needed prepare-projects e2e-deploy-latest e2e-migration-setup get-publish-and-install-operators e2e-migration-verify
```

Comment:
it looks like you already added the check as part of `get-publish-and-install-operators` , so not sure if we need to add it again here as well.

Reply:
btw, do you still face the issue after using the generated token?